flexget/plugins/clients/flood.py

```diff
@@ -0,0 +1,124 @@
+from loguru import logger
+from requests import Session
+from requests.exceptions import RequestException
+
+from flexget import plugin
+from flexget.event import event
+from requests import Session
+from loguru import logger
+
+from flexget.entry import Entry
+from flexget.task import Task
+
+logger = logger.bind(name='flood')
+
+class FloodClient:
+    def __init__(self):
+        self.session = Session()
```

Comment:
better to set trust_env to false to avoid Session using proxy environment variables

Reply:
Sorry for the late reply, but I can't find `trust_env` being used anywhere else in Flexget. Unless there's a particular reason for it that I'm not aware of?

Reply:
Personally I would not bother with trust_env.
You can read more about its usage [requests.trust_env
](https://docs.python-requests.org/en/latest/api/#requests.Session.trust_env)
It's a requests options for session to disable proxy usage. I don't see this being much of an issue, I also don't remember any other plugin using this setting during configuration. If such a setting is warranted then it should be applied to the Session wrapper currently configured in [Flexget Request](https://github.com/Flexget/Flexget/blob/develop/flexget/utils/requests.py). Usually this is the request wrapper we use in flexget as there is a domain limiting controls, cookie injections, ect.